app/javascript/articles/__stories__/Article.stories.jsx

```diff
@@ -0,0 +1,239 @@
+import { h } from 'preact';
```

Comment:
The [Storybook](https://www.learnstorybook.com) stories for the `<Article />` component.